webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -1,26 +1,70 @@
 import React from 'react';
-import { IntlProvider } from 'react-intl';
+import { IntlProvider, FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
 import { translate as __ } from 'foremanReact/common/I18n';
 import { Button } from 'patternfly-react';
 import { INSIGHTS_SYNC_PAGE_TITLE } from './InsightsCloudSyncConstants';
 
-const InsightsCloudSync = ({ syncInsights }) => {
+const InsightsCloudSync = ({ data: { settingsUrl }, syncInsights }) => {
   document.title = INSIGHTS_SYNC_PAGE_TITLE;
   return (
     <IntlProvider locale={navigator.language}>
       <div className="insights-cloud-sync">
         <h1>{__('Red Hat Insights Sync')}</h1>
-        <Button bsStyle="primary" onClick={syncInsights}>
-          {__('Sync now')}
-        </Button>
+        <p>
+          {__(`Insights synchronization process is used to provide Insights
+           recommendations output for hosts managed here`)}
+        </p>
+        <p>
+          <FormattedMessage
+            id="rh_cloud.insights.token_hint"
+            description="Hint where to obtain API token"
+            defaultMessage={`In order to set up the sychronization, you should obtain an RHSM API token
+              from {apiLink}`}
+            values={{
+              apiLink: (
+                <a
+                  href="https://access.redhat.com/management/api"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  access.redhat.com
+                  <i className="fas fa-external-link-alt" />
```

Comment:
can you use the `Patternfly Icon` component for it? e.g: `<Icon name="external-link" />`
and can you make it a bit smaller?